en-US/firefox/all.lang

```diff
@@ -74,6 +74,10 @@ Thanks to contributions of Mozilla community members around the world, Firefox i
 <strong>Warning</strong>: Nightly is updated daily. It is most appropriate for core Mozilla contributors and early adopters, not regular users.
 
 
+;<a href="%(url)s">Mozilla no longer provides security updates for Firefox on Windows XP or Vista</a>, but you can still download the final Windows 32-bit version below.
```

Comment:
This string is already available in a shared file, you don't need it duplicated here.

Reply:
Weir, I can't find it in transvision but search in VM pointed to the right place.

Reply:
How can Transvision find a string that is not available yet in the repository?